gdal/frmts/netcdf/netcdfdataset.cpp

```diff
@@ -3160,6 +3160,17 @@ void netCDFDataset::SetProjectionFromVar( int nVarId, bool bReadSRSOnly )
             }
 
 
+            // If the data is bottom up then flip the y - coords
+            // this is significantly faster than reading row by row
+            if (poDS->bBottomUp)
+            {
+                double temp = yMinMax[0];
+                yMinMax[0] = yMinMax[1];
+                yMinMax[1] = temp;
+                poDS->bBottomUp = false;
```

Comment:
A CPLDebug trace is needed here since we trace each time we set this variable. "set bBottomUp=false because GDAL_NETCDF_EXPOSE_AS=BOTTOMUP". This message is rather paradoxical actually shows that the bBottumUp variable is probably not ideally named. Perhaps bYAxisIOInversion would reflect more its effect.